### ISSUE TITLE
compile errors for MicromlMorseIdentificationExample

### ISSUE THREAD

**Issue by Riccardo451** [eloquentarduino/EloquentArduino#18]
Trying to compile the morse example file I get several errors, I've been able to compile after the following changes: 
- in model.h I've commented out -> #include <cstdarg>, and I've added instead ->  #include <stdarg.h> .

In the .ino file:
- changed -> double features[NUM_SAMPLES];     
              to -> float features[NUM_SAMPLES];
- I've commented out -> self_test(); 
- added after variable declaration -> Eloquent::ML::Port::RandomForest clf; 
-In void loop changed the serial print statement to -> Serial.println(clf.idxToLabel(clf.predict(features)));

[ml_cw_decoder.ino.txt](https://github.com/eloquentarduino/EloquentArduino/files/5780637/ml_cw_decoder.ino.txt)
[model.h.txt](https://github.com/eloquentarduino/EloquentArduino/files/5780639/model.h.txt)

**Comment by agrimagsrl** [comment]
Yeah, I know the older posts don't match with the current API. I need to take time to review all of them, sorry.
